src/test/java/org/broadinstitute/hellbender/IntelDeflaterTest.java

```diff
@@ -0,0 +1,24 @@
+package org.broadinstitute.hellbender;
+
+import htsjdk.samtools.util.zip.DeflaterFactory;
+import org.apache.commons.lang3.SystemUtils;
+import org.testng.Assert;
+import org.testng.SkipException;
+import org.testng.annotations.Test;
+
+/**
+ * Test that it's possible to load libIntelDeflater
+ */
+public class IntelDeflaterTest {
+
+    @Test
+    public void testIntelDeflaterIsAvailable(){
+        if ( !SystemUtils.IS_OS_LINUX ) {
+            throw new SkipException("IntelDeflater not available on this platform");
```

Comment:
@droazen I think this check may fail on POWER systems, but I'm not sure what the test condition should be.